From a7c8981ba50d7709418fd942866b0c57a6e37331 Mon Sep 17 00:00:00 2001
Message-Id: <a7c8981ba50d7709418fd942866b0c57a6e37331.1575314676.git.liu.denton@gmail.com>
In-Reply-To: <cover.1575314676.git.liu.denton@gmail.com>
References: <cover.1574882812.git.liu.denton@gmail.com>
	<cover.1575314676.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Wed, 27 Nov 2019 10:38:59 -0800
Subject: [PATCH v2 3/3] doc: replace LKML link with lore.kernel.org
To: Git Mailing List <git@vger.kernel.org>
Cc: Jeff King <peff@peff.net>,
    Konstantin Ryabitsev <konstantin@linuxfoundation.org>,
    Eric Wong <e@80x24.org>,
    Junio C Hamano <gitster@pobox.com>

Since we're now recommending lore.kernel.org, replace LKML link
with lore.kernel.org.

Although LKML has been around for a long time, nothing lasts forever
(see Gmane). Since LKML uses opaque message identifiers, switching to
lore.kernel.org should be a strict improvement since, even if
lore.kernel.org goes down, the Message-ID will allow future readers to
look up the referenced messages on any other archive.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 Documentation/technical/racy-git.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/technical/racy-git.txt b/Documentation/technical/racy-git.txt
index 4a8be4d144..ceda4bbfda 100644
--- a/Documentation/technical/racy-git.txt
+++ b/Documentation/technical/racy-git.txt
@@ -51,7 +51,7 @@ of git://git.kernel.org/pub/scm/linux/kernel/git/tglx/history.git
 only fixes the issue for file systems with exactly 1 ns or 1 s
 resolution. Other file systems are still broken in current Linux
 kernels (e.g. CEPH, CIFS, NTFS, UDF), see
-https://lkml.org/lkml/2015/6/9/714
+https://lore.kernel.org/lkml/5577240D.7020309@gmail.com/
 
 Racy Git
 --------
-- 
2.24.0.504.g3cd56eb17d

